adam/learner/objects.py

```diff
@@ -296,7 +297,7 @@ def get_objects_from_perception(
                     if not isinstance(neighbor, ObjectPerception):
                         other_nodes.append(neighbor)
 
-            subgraph = networkx_utils.subgraph(
+            subgraph = networkx_utils.subgraph(  # pylint: disable=redefined-outer-name
```

Comment:
Is this in reference to the `subgraph` variable? If so can you rename it to avoid this redefinition?